Extensions/outbox.js

```diff
@@ -177,7 +177,9 @@ XKit.extensions.outbox = new Object({
 
 		var form_key = XKit.interface.form_key();
 
-		$(document).on("click", "#ask_button", function() {
+		$(document).on("click", ".post-form--asks .ask-button", function() {
+
+			var $form = $(this).closest('form');
```

Comment:
right, but parent("form") will break immediately if something changes, while closest("form") may or may not return the same element we're expecting it to.

anyway, we don't need the form to find the elements we want. just use a $this variable instead.